src/parseTools.js

```diff
@@ -1292,7 +1292,7 @@ function makeGetTempRet0() {
 }
 
 function makeSetTempRet0(value) {
-  return RELOCATABLE ? "setTempRet0((" + value + ") | 0)" : ("tempRet0 = " + value);
+  return 'asm["setTempRet0"](' + value + ')';
```

Comment:
why was `RELOCATABLE` removed? I guess because both code paths work using the `asm` object?